config.cfg

```diff
@@ -0,0 +1,4 @@
+DEBUG = True
```

Comment:
This file should not be under version control. Please remove it from your pull reqeust.